src/Javis.jl

```diff
@@ -698,6 +699,7 @@ function set_object_defaults!(object)
     scaleto(desired_scale)
 end
 
+#todo add stroke/fills here
 const LUXOR_DONT_EXPORT = [
     :boundingbox,
     :Boxmaptile,
```

Comment:
It shouldn't export:
- `strokepath`
- `strokepreserve`
- `fillpath`
- `fillpreserve`
otherwise compiling is broken as a method gets overwritten

Reply:
if we dont export them, wont users have to always use 
`Luxor.strokepath()` instead of just `strokepath()`?

Reply:
You need to add them to the list in line 750/751

Reply:
i get undefvarerror when i do that